File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        
        int ct = 0;
        int maxi = 0;
        for( int i=0; i<nums.length; i++ ) {
            if( nums[i] == 1 ) {
                ct++;
                maxi = Math.max(maxi,ct);
            }
            else {
                ct = 0;
            }
        }

        return maxi;
    }
}

File: Data Structures & Algorithms/longest-substring-without-duplicates/submission-0.cpp
class Solution {
public:
    int lengthOfLongestSubstring(string s) {
        

        unordered_map<char,int> mpp;

        int maxi = 0 , ct =0;
        for( int i=0; i<s.length(); i++ ) {
            mpp.clear();
            ct =0;
            for( int j=i; j<s.length(); j++ ) {

                mpp[s[j]]++;
                if( mpp[s[j]] == 1  ) {
                    ct++;
                    maxi = max( maxi ,ct  );
                }
                else break;
            }
        }
        return maxi;
    }
};
